ethpm/utils/registry.py

```diff
@@ -0,0 +1,48 @@
+from urllib import parse
+
+from typing import List
+
+from ethpm.exceptions import UriNotSupportedError
+
+from ethpm.validation import (
+    validate_package_name,
+    validate_registry_uri_authority,
+    validate_registry_uri_scheme,
+    validate_registry_uri_version,
+)
+
+
+def is_ens_domain(auth: str) -> bool:
+    """
+    Returns false if URI authority is not a valid ENS domain.
+    """
+    if auth[-4:] != '.eth' or len(auth.split('.')) not in [2, 3]:
```

Comment:
This is sufficiently cryptic to read that it deserves a comment.  I am still not sure what the second check is validating.